src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--29 calendar--start-day--fri">
```
Потрібно реалізувати за допомогою scss циклів `@for` та `@each` можливість задавати різну кількість днів в календарі і початковий день за допомогою модифікаторів класі (як я навів у прикладі вище)

Reply:
исправил в следующем коммите